values.yaml

```diff
@@ -160,6 +160,19 @@ fleetscheduler:
       - --page-size
       - "512"
 
+# Enable the push gateway to collect metrics from fleetscheduler
+#prometheus-pushgateway:
+#  fullnameOverride: fleet-pushgateway
+#  resources:
+#    limits:
+#      cpu: 300m
+#      memory: 150Mi
+#    requests:
+#      cpu: 300m
+#      memory: 150Mi
+#  service:
+#    port: 9090
```

Comment:
Are the gateway values suppose to be commented out?

Reply:
yep, or it gets deployed by helm - theres no enable/disable bool for it